### DISCUSSION TITLE
Update fürs Movement

### DISCUSSION THREAD

**Discussion by FlorianHeck2** [FlorianHeck2/codespaces-jupyter#5] [General]
Inzwischen habe ich, neben kleinen Spielereien an der Aufgabe aus dem Unterricht, mein Movement soweit von Buggs befreit und auch der Sichtkegel (wenn auch noch nicht perfekt gezeichnet ;D ) bewegt sich mit dem Spielercharakter. Hier habt ihr zwei Screenshots, die das ganze beispielhaft zeigen!
(Man sieht, dass sich der Charakter und Sichtkreis bewegt haben.)
<img width="958" alt="image" src="https://github.com/FlorianHeck2/codespaces-jupyter/assets/152873399/86ea59c0-0f28-429a-b9d5-2958a7d629ef">

<img width="960" alt="image" src="https://github.com/FlorianHeck2/codespaces-jupyter/assets/152873399/697e81b6-4ddf-4e15-bec7-7e8123eaf638">

**Comment by Evolinox** [comment]
Sieht doch schon sehr vielversprechend aus :) Hast du schon was für die nächsten Wochen geplant?

**Reply by FlorianHeck2** [reply]
Ja, steht in der Nachricht drunter

**Comment by theresita28** [comment]
Das sieht sehr gut aus. Was sind deine nächsten Schritte?

**Reply by FlorianHeck2** [reply]
Als nächstes möchte ich nen Labyrinth basteln. Erstmal nur als Vierecke ohne schöne Grafik, damit ein erstes Level entsteht und einen Zielbereich einbauen, damit sich das Spielen lohnt. Danach mal schauen (Eigentlich wäre es dann Zeit für etwas Grafiken, aber das ist nicht so meine Stärke...

**Comment by Rouge2123** [comment]
Hastest du irgendwelche Schwierigkeiten gehabt den Feature auszubauen? Oder lief es alles ziemlich wie gedacht?

**Reply by FlorianHeck2** [reply]
Hey, es hat eigentlich gar nichts funktioniert, wie es sollte und hat recht lange gedauert.
-Das laufen an sich war kein Problem, die Hindernisse aber schon. Die meißte Zeit war es so, dass man, wenn man dagegen gelaufen ist (je nach Spielversion), nicht mehr weck konnte, oder sich die Steuerung invertiert hat...
-Bis sich die Grafik für den Sichtbereich an der richtigen Stelle geladen hat, hat es auch ein gutes bisschen gedauert, war halt etwas ausprobieren.

Aber mit Problemen muss man ja immer rechnen, also irgendwie wie gedacht
